code/modules/mob/living/carbon/human/death.dm

```diff
@@ -19,6 +19,7 @@
 /mob/living/carbon/human/death(gibbed)
 	if(stat == DEAD)
 		return
+	playsound(src, 'sound/voice/human/death.ogg', 50, 1)

```

Comment:
Now what ya SHOULD do is set the death sound as a mob variable so it can be changed via various memes.